R/url.R

```diff
@@ -0,0 +1,33 @@
+
+renv_url_parse <- function(url) {
```

Comment:
It would make me feel happier if this had a few tests.

Reply:
Added a couple -- should I try to chase down any edge cases?